Bottom: a714bc4ff96a4b6832ac8f7cb9d7f0d37f8320e7
Top:    b818a49f265b98fc20de6ebcd8b46a3d2e72a1d8
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-17 14:02:45 -0700

librdmacm: Enable AF_IB support

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/cma.c b/src/cma.c
index 5280620..9286e81 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -185,9 +185,6 @@ static void ucma_set_af_ib_support(void)
 	struct sockaddr_ib sib;
 	int ret;
 
-	/* Just return - do not enable AF_IB support for now */
-	return;
-
 	ret = rdma_create_id(NULL, &id, NULL, RDMA_PS_IB);
 	if (ret)
 		return;
